google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriter.java

```diff
@@ -762,8 +806,7 @@ private void verifyNotClosed() {
    * <p>For example, see the sample code: <code>
    *   BulkWriter bulkWriter = firestore.bulkWriter();
    *   bulkWriter.addWriteResultListener(
-   *         new WriteResultCallback() {
-   *           public void onResult(DocumentReference documentReference, WriteResult result) {
+   *         (DocumentReference documentReference, WriteResult result -> {
```

Comment:
Missing ")".

We should make sure these compile as users are going to copy them.

Reply:
Added. Checked that code compiles.